pkg/controller/kieapp/webconsole/webconsole.go

```diff
@@ -0,0 +1,102 @@
+package webconsole
+
+import (
+	"errors"
+	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/gobuffalo/packr/v2"
+	"github.com/kiegroup/kie-cloud-operator/pkg/controller/kieapp/logs"
+	creator "github.com/kiegroup/kie-cloud-operator/pkg/controller/kieapp/webconsole/creator"
+	"github.com/kiegroup/kie-cloud-operator/pkg/controller/kieapp/webconsole/factory"
+	"strings"
+)
+
+var log = logs.GetLogger("kieapp.webconsole")
+
+func LoadWebConsoleEnrichmentUI() {
+	files, err := loadFilesOnlyWithBox("consoleyamlsamples", "noop", "noop")
+	if err != nil {
+		log.Error(err, "yaml stings not loaded")
+	} else {
+		_, err = applyMultipleWebConsoleYamls(files)
+		if err != nil {
+			log.Error(err, "web console yamlsamples not successfully applied")
+		}
+	}
+	files, err = loadFilesOnlyWithBox("webconsole", "noop", "noop")
+	if err != nil {
+		log.Error(err, "yaml stings not loaded")
+	} else {
+		_, err = applyMultipleWebConsoleYamls(files)
+		if err != nil {
+			log.Error(err, "web console enrichments not successfully applied")
+		}
+	}
+}
+
+func applyMultipleWebConsoleYamls(yamlString []string) (map[int]string, error) {
```

Comment:
I'm not sure why we need all of this. It's 1 line to create an object, and an extra 3-4 lines to handle the error. For example, see [here](https://github.com/kiegroup/kie-cloud-operator/blob/master/pkg/controller/kieapp/kieapp_controller.go#L682-L686).

You just need to create an object type based on the existing yaml structure that the code can then create.

Reply:
These are all here is meant to deal with a bunch of yaml files passed in, and we don't know exactly what type they are (unless we read the file and get the kind), they could be yaml sample, consolelink, consoleexternaloglink etc.
All these are used to find the object type and then delegate to the right creator to create.

Reply:
Yes, I am not aware of any different type of yaml that needs to be handled in this code, or a situation where the caller is passing in a yaml and doesn't know what it is. Let's make this quick and simple, to handle the only types we need to create here, which is CR samples.

Reply:
That means we don't need the factor and creator. Modified.